pos_lot_selection/static/src/js/popup/EditListPopup.js

```diff
@@ -0,0 +1,20 @@
+/*
+    Copyright 2022 Camptocamp SA
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+*/
+odoo.define("pos_lot_barcode.EditListPopup", function (require) {
+    "use strict";
+
+    const EditListPopup = require("point_of_sale.EditListPopup");
+    const Registries = require("point_of_sale.Registries");
+
+    const LotSelectEditListPopup = (EditListPopup) =>
+        class extends EditListPopup {
+            constructor() {
+                super(...arguments);
+                this.props.lots = this.env.session.lots;
```

Comment:
I guess we could do this to make sure the value is set only if we are on the `EditListPopup` for lot selection. (in case this Popup is used in another module)
```suggestion
                if (this.props.title === this.env._t("Lot/Serial Number(s) Required")) {
                    this.props.lots = this.env.session.lots;
                }
```